var/spack/repos/builtin/packages/fpm/package.py

```diff
@@ -27,6 +27,7 @@ class Fpm(Package):
     variant("openmp", default=True, description="Use OpenMP parallelisation")
 
     depends_on("curl", type="build")
+    depends_on("git@:1.8.5", type="build")
```

Comment:
Doe this package depend on _old_ versions of git? :worried:

Reply:
The other way round it is:
```suggestion
    depends_on("git@1.8.5:", type="build")
```

Reply:
Whoops you're right, I guess I need more coffee.